README.md

```diff
@@ -161,6 +166,13 @@ to administer Fluo:
 If you run some tests and then want a fresh cluster, run the `setup` command again which will
 kill all running processes, clear any data and logs, and restart your cluster.
 
+## Wipe command
+
+The `wipe` command will kill all running processes for your local development cluster and clear
+all the data and logs. It does *not* delete the binary tarballs downloaded by the `fetch` command
```

Comment:
Could mention that the data and logs are cleared from the Install directory.

Reply:
internal details should be left out, per conversation in review